projects/mercury/src/file/FileAPI.js

```diff
@@ -479,12 +472,6 @@ class FileAPI {
     mapToFile = fileObject => {
         const properties = {...fileObject, ...(fileObject.props || {})};
         delete properties.props;
-        Object.keys(properties).forEach(key => {
-            // The WebDAV client does not properly decode the XML response,
```

Comment:
Let me be honest, I still don't understand this part of the code. Did you check the problem this piece of code solves? Don't we remove a fix of something?

Reply:
I mean I see your comment, I saw the ticket you created. Does this ticket cover the issue? How critical is it?

Reply:
good, let's indeed remove it and see how it goes